README.md

```diff
@@ -9,3 +9,10 @@ Uses blackfriday to process markdown into man pages.
 ### Usage
 
 ./md2man -in /path/to/markdownfile.md -out /manfile/output/path
+
+### How to contribute
+
```

Comment:
I tried to describe govend in `vendor/README.md`, but `govend -v -u --prune` automatically removes extra files under the `vendor` directory.
So I edited the top-level `README.md`